Resources/Prototypes/Entities/Objects/Misc/subdermal_implants.yml

```diff
@@ -27,6 +27,7 @@
       mobState:
       - Dead
     - type: EmitSoundOnTrigger
+      predicted: true
```

Comment:
```suggestion
      predicted: false
```
Seems you can't predict here since this goes through an implant relay, which other clients won't know about.

Reply:
Wait you use PlayPredicted not PlayLocal so uhhhh I don't know why this fixes this (yet)